### ISSUE TITLE
LOGO ERROR

### ISSUE THREAD

**Issue by Manoj-N00** [aadltya/cwm-docs#1]
error rendering ur logo
![image](https://github.com/user-attachments/assets/da2ac99f-1456-4513-995c-d6cfea2a0df2)

**Comment by aadltya** [comment]
yes i know still lot to add in docs like kubernetes, docker, postgreSQL, typescript,  some CP and more

**Comment by aadltya** [comment]
refer pull request #2

**Comment by aadltya** [comment]
> error rendering ur logo ![image](https://private-user-images.githubusercontent.com/174461703/370810521-da2ac99f-1456-4513-995c-d6cfea2a0df2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU0NTQ5ODIsIm5iZiI6MTczNTQ1NDY4MiwicGF0aCI6Ii8xNzQ0NjE3MDMvMzcwODEwNTIxLWRhMmFjOTlmLTE0NTYtNDUxMy05OTVjLWQ2Y2ZlYTJhMGRmMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyOVQwNjQ0NDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lYTJiYmFmYjY1ZmYyNjQ3YmE1MzU2ZWJlODUwODZjZTg0YzM1Y2U2OWI3NTA4NDkwNjM4YTA0ODRkZjk5OWFhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.v-M0G6ToGjQqCrQLUP2Hqm_07b61qdZ317jz6qm1pCI)

refer pull request #2

**Comment by aadltya** [comment]
refer pull request #2
